src/stores/ClientPresenceStore.js

```diff
@@ -40,6 +40,8 @@ class ClientPresenceStore extends PresenceStore {
           large_image: assets.get(activity.assets.largeImage) || activity.assets.largeImage,
           small_image: assets.get(activity.assets.smallImage) || activity.assets.smallImage,
         } : undefined,
+        timestamps: activity.timestamps || undefined,
```

Comment:
This should convert the Date object back to a unix timestamp.

Reply:
not needed